src/l10n/fr.json

```diff
@@ -0,0 +1,116 @@
+        "Never"                                                                                     : "Jamais",
+        "Immediately"                                                                               : "Immédiatement",
+        "After two hours"                                                                           : "Après deux heures",
+        "After one day"                                                                             : "Après une journée",
+        "After two weeks"                                                                           : "Après deux semaines",
+        "After one month"                                                                           : "Après un mois",
+        "After one year"                                                                            : "Après un an",
+        "Caches"                                                                                    : "Caches",
+        "Default Cache (%s files, %s)"                                                              : "Cache par défaut (%s fichiers, %s)",
+        "Avatars Cache (%s files, %s)"                                                              : "Avatars (%s fichiers, %s)",
+        "Favicon Cache (%s files, %s)"                                                              : "Favicon (%s fichiers, %s)",
+        "Preview Cache (%s files, %s)"                                                              : "Aperçus (%s fichiers, %s)",
+        "Passwords Cache (%s files, %s)"                                                            : "Mots de passe (%s fichiers, %s)",
+        "clear"                                                                                     : "Purger",
+        "Clear this cache"                                                                          : "Purger ce cache",
+        "You can not clear a cache using a shared service"                                          : "Vous ne pouvez pas vider ce cache tant que le service utilise une instance d’API partagée.",
+        "Saved"                                                                                     : "Sauvé",
+        "Cleared"                                                                                   : "Purgé",
+        "Failed"                                                                                    : "Échec",
+        "Need help"                                                                                 : "Besoin d’aide ?",
+        "found a bug"                                                                               : "Bogue trouvé ?",
+        "looking for the documentation"                                                             : "vous cherchez la documentation (anglais uniquement)",
+        "%s, %s or %s? We've got you covered!"                                                      : "%s, %s ou %s ? Toute l’aide est par ici",
```

Comment:
Lines 67 - 70 are used at the end of the admin settings in one sentence. Currently the result is ` Besoin d’aide ?, Bogue trouvé ? ou vous cherchez la documentation (anglais uniquement) ? Toute l’aide est par ici`. Is that correct?

Reply:
If I understand well, the `%s` in the line 70 are respectively replaced by lines 67 to 69, am I correct ?

If that's the case, then you should either not put punctuation in the messages or remove punctuation in the main sentence.
So it would be either of the following solutions: 
```
        "Need help"                                                                                 : "Besoin d’aide",
        "found a bug"                                                                               : "Bogue trouvé",
        "looking for the documentation"                                                             : "Vous cherchez la documentation (anglais uniquement)",
        "%s, %s or %s? We've got you covered!"                                                      : "%s ? %s ? %s ? Toute l’aide est par ici",
```

```
        "Need help"                                                                                 : "Besoin d’aide",
        "found a bug"                                                                               : "bogue trouvé",
        "looking for the documentation"                                                             : "vous cherchez la documentation (anglais uniquement)",
        "%s, %s or %s? We've got you covered!"                                                      : "%s, %s ou %s ? Toute l’aide est par ici",
```

Whatever suits you best 😉